cms/djangoapps/course_to_library_import/views/v0/tests/test_views.py

```diff
@@ -4,29 +4,63 @@
 
 from unittest import mock
 
-from django.test import TestCase
 from django.urls import reverse
+from organizations.models import Organization
 from rest_framework.test import APIClient
+from rest_framework import status
 
-from cms.djangoapps.course_to_library_import.data import CourseToLibraryImportStatus
-from cms.djangoapps.course_to_library_import.tests.factories import CourseToLibraryImportFactory
 from common.djangoapps.student.tests.factories import UserFactory
+from cms.djangoapps.course_to_library_import import api
+from cms.djangoapps.course_to_library_import.constants import COURSE_TO_LIBRARY_IMPORT_PURPOSE
```

Comment:
Should it be a target naming instead of purpose?
Like `COURSE_TO_LIBRARY_IMPORT_TARGET`

Reply:
I think this is a rather global issue. I propose to leave it as it is for now. 
I will come back to this later after discussing it with Dave.